src/ngx_pagespeed.cc

```diff
@@ -3019,13 +3019,17 @@ ngx_int_t ps_preaccess_handler(ngx_http_request_t* r) {
   ph = cmcf->phase_engine.handlers;
 
   i = r->phase_handler;
+
   // move handlers before try_files && content phase
+  // As of nginx 1.13.4 we will be right before the try_files module
+  #if (nginx_version < 1013004)
```

Comment:
I'd put 1013004 into a const int so (a) you can doc what that release number means in the constant name and (b) you are referencing it below as well.